src/index.js

```diff
@@ -6,68 +6,105 @@ import reportWebVitals from './reportWebVitals';
 import { config } from './constants'
 import axios from 'axios';
 
-// class Price extends React.Component {
-//   constructor(props) {
-//     super(props);
-//     this.state = {
-//       isLoaded: false,
-//       stocks: [],
+class Price extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isLoaded: false,
```

Comment:
Is `isLoaded` being used?

Reply:
It isn't being used at the moment, but I'd like to leave it in as a useful tool for later (might be helpful for components down the line as I learn more about React). For now I'll comment it out.